test/ppspp_have_SUITE.erl

```diff
@@ -35,5 +35,6 @@ have(_Config) ->
     Method = chunk_32bit_chunks,
     Garbage = crypto:strong_rand_bytes(10),
     Chunks = <<15:64, Garbage/binary>>,
-    Expected = {{have,{chunk_spec,{chunk_32bit_chunks,0,15}}}, Garbage},
+    %% Expected = {{have,{chunk_spec,{chunk_32bit_chunks,0,15}}}, Garbage},
+    Expected = { #{have => #{chunk_spec => {chunk_32bit_chunks,0,15} }}, Garbage},
```

Comment:
Line 39 is too long:     Expected = { #{have => #{chunk_spec => {chunk_32bit_chunks,0,15} }}, Garbage},.